### PR TITLE
Add Vet Centers to Facility Locator

### PR DESCRIPTION
**Pull request by bshyong**
Resolve https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3160

**Do NOT merge until Vet Centers are ready to launch**

This PR adds Vet Centers to the search controls dropdown on facility locator

### CODE REVIEW

File: src/js/facility-locator/components/SearchControls.jsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ class SearchControls extends Component {
     const { currentQuery: { facilityType } } = this.props;
 
     switch (facilityType) {
-      case 'benefits':
+      case 'va_benefits_facility':
```
Comment (U-DON) [inline src/js/facility-locator/components/SearchControls.jsx]: Should this be a switch if there is only one specified case and a default? Seems more fitting to just use if/else.

File: src/js/facility-locator/components/SearchControls.jsx [hunk 2]
```diff
@@ -125,32 +125,28 @@ class SearchControls extends Component {
...
+      va_cemetery: 'cemetery',
+      va_benefits_facility: 'benefits',
+      vet_center: 'vet-center',
+    };
+    /* eslint-enable camelcase */
+
+    if (facilityType) {
+      return (
+        <button type="button" className="facility-option">
+          <span className="flex-center"><span className={`legend ${classMappings[facilityType]}-icon`}></span>{facilityTypes[facilityType]}</span>
```
Comment (U-DON) [inline src/js/facility-locator/components/SearchControls.jsx]: Seems like `classNames` could be used here.

```
const legendClass = classNames(
  'legend',
  `${classMappings[facilityType]}-icon`
);
```

Also could potentially avoid the `eslint-disable` with something like:


```
const legendClass = classNames(
  'legend',
  { 'health-icon': facilityType === 'va_health_facility' },
  { 'cemetary-icon': facilityType === 'va_cemetary' },
  ...
);
```

File: src/js/facility-locator/components/SearchControls.jsx [hunk 3]
```diff
@@ -197,9 +193,10 @@ class SearchControls extends Component {
               </div>
               <ul className="dropdown">
                 <li onClick={this.handleFacilityFilterSelect()}>{this.renderSelectOptionWithIcon()}</li>
-                <li onClick={this.handleFacilityFilterSelect('health')}>{this.renderSelectOptionWithIcon('health')}</li>
-                <li onClick={this.handleFacilityFilterSelect('benefits')}>{this.renderSelectOptionWithIcon('benefits')}</li>
-                <li onClick={this.handleFacilityFilterSelect('cemetery')}>{this.renderSelectOptionWithIcon('cemetery')}</li>
+                <li onClick={this.handleFacilityFilterSelect('va_health_facility')}>{this.renderSelectOptionWithIcon('va_health_facility')}</li>
```
Comment (U-DON) [inline src/js/facility-locator/components/SearchControls.jsx]: I wonder if these could be stored as constants since they show up a lot.

**Review by U-DON — APPROVED** [review]
LGTM. Left some minor comments.

**Comment by bshyong** [comment]
Good comments—would like to clean FL up and refactor after both vet centers and access to care are launched. The move to camelcase and some other clean up is on the access to care branch

**Pull request closed without merging**